docs/diagrams/list/ListActivityDiagram.puml

```diff
@@ -0,0 +1,12 @@
+@startuml
+start
+:User executes command;
+:ListCommandParser parses the arguments;
+if () then ([Syntax error] )
+    :"Show UsageSyntax";
+else ( [correct input])
+    :"Create new ListCommand object";
+endif
+:Show filtered list of Transactions;
```

Comment:
Put this inside the `else` clause, `Syntax error` should not show the filtered list.

Reply:
Pointed out by @lightz96.